app.js

```diff
@@ -1 +1,100 @@
- 
\ No newline at end of file
+// Book Array to hold books
+const myLibrary = [
+  {
+    title: 'Default One',
+    author: 'John Doe',
+    pages: 1000,
+    read: false,
+  }, {
+    title: 'Default Two',
+    author: 'Jane Doe',
+    pages: 1500,
+    read: false,
+  }];
+
+// Book function
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+// Function to add books to View
+function addBookToLibrary(book) {
+  const list = document.querySelector('#book-table');
+  const row = document.createElement('tr');
+  book.read = document.getElementById('checkbox').checked;
+  const state = book.read === true ? 'Yes I have' : 'No I have not';
+
+  row.innerHTML = `
+    <td>${book.title}</td>
+    <td>${book.author}</td>
+    <td>${book.pages}</td>
```

Comment:
- [ ] [OPTIONAL] Let's use some object destructuring instead of repeating `book.` so many times.

You can declare each book field you need like: 

`const {title, author, page} =  book`

at the top and then you'll be able to drop the `.book`;